src/Routing/Next.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Tnapf\Router\Routing;
+
+use LogicException;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use stdClass;
+use Tnapf\Router\Interfaces\RequestHandlerInterface;
+
+class Next
```

Comment:
Is this just for running the next middleware\postware in the stack? Seems massively overcomplicated if so

Reply:
It stores then runs the next piece, yes. I can't think of a better way to simplify it. I was however thinking of adding an `__invoke` magic function so they could just invoke the instance instead of doing `Next::next()`

Reply:
Copy the implementation from Bifrost, essentially the same. Far more concise